client/src/component/DOS/DefineAudit/SelectContestsPageContainer.tsx

```diff
@@ -58,10 +58,20 @@ class SelectContestsPageContainer extends React.Component<ContainerProps> {
 }
 
 function mapStateToProps(dosState: DOS.AppState) {
-    const isAuditable = (contestId: number): boolean => {
+    const isAuditable = (contest: Contest): boolean => {
         if (!dosState.auditTypes) { return false; }
 
-        const t = dosState.auditTypes[contestId];
+        if (contest.description === 'IRV') {
+            const assertionSummary = dosState.generateAssertionsSummaries.find(
+                element => element.contestName === contest.name);
+
+            // Tied IRV contests are not auditable
+            if (assertionSummary && assertionSummary.error === 'TIED_WINNERS') {
```

Comment:
There might be other kinds of errors (though tied winners is the only one likely to arise in practice), so it would be better if this just checked that the error was not blank/empty.

If you give it a ridiculously small time limit, you might be able to get a TIME_OUT_GENERATING_ASSERTIONS error from one of the New South Wales cases (e.g. Ballina).